### PR TITLE
Politician page css

### PR DESCRIPTION
**Pull request by AlbinoArtist**
# Main points

- Created a sound SASS structure 
- Added some css styling to politician page.
- Changed the html and look of the politician page.

# Future additions

- Should change color dynamically depending on politician party. 
- Need to add links and icons for facebook, g+, twitter etc.

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Component } from '@angular/core';
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  styleUrls: ['./app.component.scss',]
```
Comment (fbontin) [inline src/app/app.component.ts]: Maybe a typo? Or there might be something about typescript/javascript I don't know yet and this does something :-)

File: src/app/politician-page/politician-page.component.html [hunk 2]
```diff
@@ -1,28 +1,44 @@
...
+          <h3>{{Politician.status}}</h3>
+          <h3>{{Politician.parti}}</h3>
+        </div>
+      </div>
+
+      <div class="row politician-info">
+        <!--Politician contact info-->
+        <div class="col">
+          <ul>
+            <li><strong>Email</strong><span>{{Politician.tilltalsnamn}}.{{Politician.efternamn}}@riksdagen.se</span></li>
```
Comment (fbontin) [inline src/app/politician-page/politician-page.component.html]: Might be an unreliable way of linking to their email. That said, I did not find the email in the politician data.

Comment (AlbinoArtist) [inline src/app/politician-page/politician-page.component.html]: Just for mock data. Real email cam be pulled from the politician get response.

File: src/app/app.component.ts (hunk 1 above)
Comment (AlbinoArtist) [inline src/app/app.component.ts]: Typo

Comment (fbontin) [inline src/app/app.component.ts]: 👍

File: src/app/politician-page/politician-page.component.html (hunk 2 above)
Comment (fbontin) [inline src/app/politician-page/politician-page.component.html]: 👍

**Pull request merged**